lib/core-common/src/utils/template.ts

```diff
@@ -8,10 +9,7 @@ export function getPreviewBodyTemplate(
   configDirPath: string,
   interpolations?: Record<string, string>
 ) {
-  const base = fs.readFileSync(
-    path.resolve(__dirname, '../templates/base-preview-body.html'),
-    'utf8'
-  );
+  const base = fs.readFileSync(`${sync(__dirname)}/templates/base-preview-body.html`, 'utf8');
```

Comment:
Here's me using the `pkg-dir` `sync` command to get the `fs` location (directory) of `package.json`, to then concat that with where I know the template exists.

I did not use the `@storybook/core-common......body.html` here, because of the test. The `mock` util which I don't know very well, can't deal with that, and then `node` complains about not being able to resolve.